static/app/views/organizationGroupDetails/eventToolbar.tsx

```diff
@@ -99,7 +99,7 @@ class GroupEventToolbar extends Component<Props> {
   }
 
   render() {
-    const is24Hours = use24Hours();
+    const is24Hours = shouldUse24Hours();
```

Comment:
no strong opinion here but maybe remove `use` completely? e.g. `shouldDisplay24Hours`?

Reply:
No strong opinion. I wanted to keep whatever the original intent was here without trying to understand the context too much. I'm +1 on using shouldDisplay24Hours.